### DISCUSSION TITLE
Looking for founders to make Joyboy the next unicorn of SocialFi

### DISCUSSION THREAD

**Discussion by AbdelStark** [keep-starknet-strange/joyboy#68] [Announcements]
Hey, Wanna take Joyboy to next level and make it a successful business on Starknet ? Let's chat.

Few starting points:
- Join the Telegram group: https://t.me/JoyboyStarknet
- Have a look at the Github repository for a brief explainer of the project: https://github.com/keep-starknet-strange/joyboy
- Apply here: https://form.typeform.com/to/ZlhKCqxV

Tell us more about you and if you are interested we can discuss.

**Comment by YashBit** [comment]
@AbdelStark  Interested. NYU Grad, Research in DeepRL, NLU, Cognitive Science. SWE (Few Jobs)

- Worked as EIR DeFI Startup (ERC 3643 Private Tokens)

**Comment by Israelrex9** [comment]
Applied @AbdelStark 
I am very experienced in designing very friendly interfaces and I have a vision for JoyBoy to be an open, welcoming, playful decentralized app. I once talked about streaks on the telegram group, and I believe we can work more around it to make the app more interactive, think of the highest streak as One Piece (based on the movie OnePiece the treasure Luffy and his team searched for all through)

Basically, we can gamify JoyBoy.
Happy to shed more as we build this game-changing SocialFi
